android/autodispose-androidx-lifecycle-test/src/main/java/autodispose2/androidx/lifecycle/test/KotlinExtensions.kt

```diff
@@ -24,6 +24,7 @@ import io.reactivex.rxjava3.annotations.CheckReturnValue
 /**
  * Extension that returns a [TestLifecycleOwner] for this [LifecycleRegistry].
  */
+@Deprecated("Switch to androidx.lifecycle.testing.TestLifecycleOwner")
```

Comment:
Intentionally not using `ERROR` since there's no immediate replaceWith option

Reply:
Solo pongo ERROR